src/garage/tf/embeddings/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""Embeddings."""
+from garage.tf.embeddings.base import Embedding, EmbeddingSpec
```

Comment:
please don't use a `base` package -- instead put the classes in their own files

Reply:
also -- please follow the style for this module introduced in `garage.torch.embeddings`